index.html

```diff
@@ -0,0 +1,153 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/2649a194d9.js" crossorigin="anonymous"></script>
+</head>
+<body>
+      <!-- Desktop first nav -->
+    <div class="dskNav1">
+        <ul class="dskNavUl1">
+            <li class="nav-item">
+                <a href="#" class="nav-link"><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+
+        <div class="dskNav2">
+            <div class="logoImg">
+                <img src="./images/myLogo.jpg" alt="Logo">
+                <p>Adorable Forex Trading</p>
+            </div>
+
+            <ul class="dskNavUl2">               
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#mainProgram">Program</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#speakerContent">Speakers</a></li>
+                <li><a href="#footerLogo">Partners</a></li>
+                <li class="classes"><a href="#footerLogo">Classes</a></li>
+            </ul>          
+        </div>
+    </div>
+
+    <section class="firstpageCover">  
+        <nav>               
+                <div class="hamburger">
+                    <div class="line1"></div>
+                    <div class="line2"></div>
+                    <div class="line3"></div>
+                </div>                
+                <ul class="navList" id="navLi">
+                    <li class="closeBtn">&times</li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#mainProgram">Program</a></li>
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#desktopSpeakers">Speakers</a></li>
+                    <li><a href="./about.html">Partners</a></li>
+                </ul>
+        </nav>
+        <section class="landingPage">
+                <div class="heading">
+                    <p>"Adorable Forex Trading"</p>
+                    <h1 class="image-Letter">FIRST YOU LEARN, THEN YOU TAKE OUT THE 'L'</h1>
+                </div>
+            
+                <div class="headingPara1">
+                    <P>Trading effectively is about assessing probabilities, 
+                        not certainties.” ― Yvan Byeajee</P>
+                </div>
+
+                <div class="headingPara2">
+                    <h4>8AM ~ 5PM MON ~ FRI</h4>
+                    <p>@ N0.31 Shopping Plaza, Old G.R.A, PH</p>
+                </div>
+        </section>
```

Comment:
- [ Optional ] The `font-family` used on the heading in the `jumbotron` sections of the `index.html` and `about.html` is `Cocogoose`  whereas in your design you are using `Lato` font, It would be better if your design is near the design on Behance in terms of similarity. You can import the `Cocogoose` font and use it in your `styles.css` as shown below.

```CSS
 @import url('https://fonts.cdnfonts.com/css/cocogoose');
 
.heading {
   font-family: cocogoose;
 }
```